gc/mmtk/extconf.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative "../extconf_base"
```

Comment:
Where is this file?

Reply:
This file will be part of Ruby. 

https://github.com/ruby/ruby/pull/12149

I am currently working on a `rake compile` task that will mean that we can build and test Ruby with MMTk integration entirely using the `ruby/mmtk` repository, which we'll be able to ship once GH-12149 has been merged.